src/index.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
```

Comment:
no need for `* as`, it's redundant, you can just `import React from 'react'` and so on